### ISSUE TITLE
Sharing pane not always open for CSS

### ISSUE THREAD

**Issue by renyuneyun** [solid/solidos#118]
I'm encountering this issue on CSS (Community Solid Server), but not on `solidcommunity.net` (which runs Node Solid Server, as far as I'm told).

After setting up the server using mashlib recipe, and logging in to my user account over the mashlib, I navigated to my storage.
Opening any folder works fine. When clicking on the "lock" button, it open the sharing setting pane initially.
Coming back to the folder tree, and back to the "lock" button, it no longer opens the sharing setting pane.
The same also happens when opening some other folders, clicking on the "lock" button; same is true (i.e. won't open) when moving back to the original folder.
![图片](https://user-images.githubusercontent.com/3293723/154046413-44186543-bc73-4ac6-a52a-1af08377a5cc.png)

After refreshing the webpage, I'm back to initial: can open the sharing setting panel initially, but not the second time.

When opening the sharing pane (including the "correctly opening" case), there is often a warning on the console:
```
[PermissionBasedAuthorizer] warn: Unauthenticated agent has no read permissions
```
This *often* happens, but not always. Not sure if this is related to the incorrect permission of my avatar file (which is a separate issue on its own).

**Comment by bourgeoa** [comment]
Thanks for reporting. I added it to bug issue

**Comment by bourgeoa** [comment]
@renyuneyun This should be resolved with the test-version of mashlib@1.7.20-401
It shall solve all problems related to the sharing-pane in CSS.
Can you report the findings.

**Comment by renyuneyun** [comment]
@bourgeoa Thanks for the work! I tested it, but still encounter the 401 errors when opening sharing-panel and when modifying the sharing setting.

In case I did anything wrong, I would like to check my steps in using the newer version of mashlib:
1. Clone the mashlib repo locally to `/root/mashlib/`.
2. In mashlib directory, checkout the 1.7.20-401 version: `git checkout origin/error401`
3. In mashlib directory, install dependencies and generate the `dist/` directory: `npm install --dev && npm run build`.
4. In `community-server-recipe` (cloned previously; not described in the steps here), modify `packages.json` to point to the mashlib cloned locally: `"mashlib": "file:/root/mashlib"`.
5. In recipe, reinstall dependencies and do ci: `npm install && npm ci --production`
6. Run CSS from recipe as did previously `npx community-server ARGUMENTS`.
